presto-docs/src/main/sphinx/release/release-0.237.rst

```diff
@@ -0,0 +1,61 @@
+=============
+Release 0.237
+=============
+
+General Changes
+_______________
+* Fix NPE in common sub-expression optimization triggered by ``CASE-WHEN`` expression.
+* Fix compiler failure with function type when common sub-expression optimization is enabled.
+* Improve TaskStatus JSON decoding by making it leaner.
+* Add JSON format for ``EXPLAIN`` with type ``LOGICAL`` and ``DISTRIBUTED``.
+* Add functions :func:`array_sum` and :func:`map_normalize`.
+* Add new session property ``warning_handling`` to control how warnings are handled. The options are ``SUPPRESS``, ``NORMAL`` and ``AS_ERROR``. The default value is ``NORMAL``.
+* Add support for Druid's basic and kerberos authentication.
+* Add support for defining SQL-invoked functions in plugins.
+* Add a new aggregation function :func:`set_agg` - equivalent to  ``ARRAY_AGG(DISTINCT x)``.
+* Add NodeStatusService interface to find out if a worker is in maintenance mode or ready to take tasks. Default behavior is no op and can be overridden. Config controlling the
+  behavior is ``node-status-service.enabled``.
+* Add suggestion to use :func:`approx_distinct` when using ``COUNT(DISTINCT x)``.
+* Add support for listing functions whose names match a specified pattern using the ``SHOW FUNCTION LIKE`` syntax.
+
+Cassandra Changes
+_________________
+* Fix missing Netty library introduced in version 0.229.
+
+Elasticsearch Changes
+_____________________
+* Fix predicate pushdown for Elasticsearch.
+* Improve Elasticsearch (save an extra hop) by using shard primary host in Elasticsearch connector.
+* Add Elasticsearch array support using definitions in the _meta field.
+* Add support for AWS IAM authorization to Elasticsearch connector.
+* Add support for Elasticsearch query string syntax.
+* Add support for nested types in Elasticsearch.
+* Add support for querying Elasticsearch aliases.
+* Add system.nodes table to Elasticsearch.
+* Add handling for mixed-case columns in Elasticsearch.
+* Add functionality to load tables dynamically in Elasticsearch.
+* Add functionality to refresh Elasticsearch nodes periodically.
+* Replace Elasticsearch TransportClient with Http client in Elasticsearch connector.
```

Comment:
What is the impact of this?

Reply:
cc @zhenxiao, could you please provide the impact of this change?

Reply:
this is an internal api update.

Reply:
@zhenxiao  - Should we have it in the release notes then? Generally we try to cover things that can have some user impact or sys admin impact

Reply:
we could remove this line